src/components/dev-dot-content/mdx-components/mdx-accordion/mdx-accordion.module.css

```diff
@@ -0,0 +1,3 @@
+.accordionWrapper {
+	margin-top: 16px;
+}
```

Comment:
(idea): Though this works because of margin-collapsing, what about only applying `margin-top` to accordions that immediately follow a previous accordion?

pseudo-css
```css
.accordionWrapper + .accordionWrapper {
  margin-top: 16px;
}
```

Reply:
💭 My thought is that we always want the top margin because we can't count on the previous sibling having bottom margin.